tests/datasource/test_new_datasource_with_runtime_data_connector_sparkdf_execution_engine.py

```diff
@@ -156,45 +156,6 @@ def test_batch_data_sparkdf_execution_engine_unknown_data_connector(
         )
 
 
-def test_batch_data_sparkdf_execution_engine_no_batch_identifiers(
```

Comment:
@donaldheppner I do not intuitively see why this test is being deleted. Could you please help with a justification? Thank you.

Reply:
These tests check the RuntimeBatchRequest constructor for the same logic that is tested other places and in test_batch.py -- the actual assertions here have nothing to do with Spark.